ch10.asciidoc

```diff
@@ -954,7 +954,7 @@ The two chains continue to diverge from this point. Miners on the "b" chain will
 
 ((("forks", "changing consensus rules", "software forks")))For software developers, the term "fork" has another meaning, adding confusion to the term "hard fork." In open source software, a fork occurs when a group of developers choose to follow a different software roadmap and start a competing implementation of an open source project. We've already discussed two circumstances that will lead to a hard fork: a bug in the consensus rules and a deliberate modification of the consensus rules. In the case of a deliberate change to the consensus rules, a software fork precedes the hard fork. However, for this type of hard fork to occur, a new software implementation of the consensus rules must be developed, adopted, and launched.
 
-Examples of software forks that have attempted to change consensus rules include Bitcoin XT, Bitcoin Classic, and most recently Bitcoin Unlimited. However, none of these software forks have resulted in a hard fork. While a software fork is a necessary precondition, it is not in itself sufficient for a hard fork to occur. For a hard fork to occur, the competing implementation must be adopted and the new rules activated, by miners, wallets, and intermediary nodes. Conversely, there are numerous alternative implementations of Bitcoin Core, and even software forks, that do not change the consensus rules and barring a bug, can coexist on the network and interoperate without causing a hard fork.
+Examples of software forks that have attempted to change consensus rules include Bitcoin XT, Bitcoin Classic, Bitcoin Unlimited, and most recently Bitcoin Cash. However, none of these software forks have resulted in a hard fork. While a software fork is a necessary precondition, it is not in itself sufficient for a hard fork to occur. For a hard fork to occur, the competing implementation must be adopted and the new rules activated, by miners, wallets, and intermediary nodes. Conversely, there are numerous alternative implementations of Bitcoin Core, and even software forks, that do not change the consensus rules and barring a bug, can coexist on the network and interoperate without causing a hard fork.
```

Comment:
But Bitcoin Cash did result in a hard fork?

Reply:
regarding: "Examples of software forks that have attempted to change consensus rules include Bitcoin XT, Bitcoin Classic, Bitcoin Unlimited, and most recently Bitcoin Cash" 

Bitcoin Cash was not an _attempt_ to change the consensus rules of Bitcoin. It created a successful chain split/fork, which technically was a Hard Fork with a minority of miners adopting. Bitcoin Cash should be removed from this section since it is covered elsewhere.